mysql-test/r/mysqldump.result

```diff
@@ -679,7 +679,7 @@ drop database mysqldump_test_db;
 # --default-character-set=xxx. However, we should dump in UTF8
 # if it is explicitely set.
 CREATE TABLE t1 (a  CHAR(10));
-INSERT INTO t1  VALUES (_latin1 '??');
+INSERT INTO t1  VALUES (_latin1 '����');
```

Comment:
This is some accidental change, maybe because of the editor encoding?

Reply:
This is OK. '??' is wrong caused by editor. 
This file looks like:
![image](https://user-images.githubusercontent.com/50845928/84629554-cc37a300-aeea-11ea-9b9c-c5f1bd750816.png)
and in fact is the same as upstream (PS) version of it.